core/services/ocr2/plugins/ocr2keeper/evmregistry/v20/log_provider.go

```diff
@@ -69,7 +68,9 @@ func NewLogProvider(
 
 	// Add log filters for the log poller so that it can poll and find the logs that
 	// we need.
-	err = logPoller.RegisterFilter(logpoller.Filter{
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
This is fixable. You can go to the top caller which is `newServicesOCR2Keepers` and with a few minor changes just pass the ctx instead.